cmd/kubelet/app/server.go

```diff
@@ -266,6 +263,13 @@ is checked every 20 seconds (also configurable with a flag).`,
 				KubeletConfiguration: *kubeletConfig,
 			}
 
+			if utilfeature.DefaultFeatureGate.Enabled(zpagesfeatures.ComponentFlagz) {
```

Comment:
Does this need to move above the kubeletServer initialization that's happening above?
Also if possible, could you also add a snippet of the response you get from kubelet's /flagz page after this change in the PR description? I was testing the new endpoint in a kind cluster, and added what I was getting back for the apiserver [here](https://github.com/kubernetes/kubernetes/pull/127581)

Reply:
ah I think I was looking at the older version of the code earlier. You now have added `Flagz` as a field in `kubeletServer` rather than `kubeletConfig`